<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>204</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Rename</string>
     </attribute>
     <widget class="QLineEdit" name="pathEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchPath">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Folder</string>
      </property>
     </widget>
     <widget class="QComboBox" name="actionTypeSelector">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>72</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Copy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rename</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Count Start</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="countStart">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>43</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="undo_Button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goRename_Button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Remove</string>
     </attribute>
     <widget class="QLineEdit" name="pathEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchPath_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goRemove_Button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phrase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Phrase (Optional)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undo_Button_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Undo</string>
     </attribute>
     <widget class="QLineEdit" name="pathEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchPath_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
